examples/platforms/cc2538/radio.c

```diff
@@ -103,6 +103,11 @@ void enableReceiver(void)
 
 void disableReceiver(void)
 {
+    if (!sIsReceiverEnabled)
```

Comment:
adding it or not didn't have any influence on current test result, however I think adding it here would not do any harm. 
Updated a new patch which put it within if-statement.